README.md

```diff
@@ -1 +1,246 @@
-# brainypedia-design-document
\ No newline at end of file
+# BrainKB Design Document
+<img src="logo.png" width="100px" style="float:right; margin-top:-45px"/>
+
+**Author:** Tek Raj Chhetri | <tekraj@mit.edu>
+
+## Overview
+_BrainKB serves as a knowledge base platform that provides scientists worldwide with tools for searching, exploring, and visualizing Neuroscience knowledge represented by knowledge graphs (KGs)_. Moreover, BrainKB provides cutting-edge tools that enable scientists to contribute new information (or knowledge) to the platform, ensuring it remains the go-to destination for all neuroscience-related research needs.
```

Comment:
since this is brand new platform I would avoid saying "ensuring it remains the go-to destination"

Reply:
updated - "...is expected to be a go-to destination..."